wallet/src/types.rs

```diff
@@ -94,8 +94,19 @@ impl WalletConfig {
 		format!("{}:{}", self.api_listen_interface, self.api_listen_port)
 	}
 
+	// TODO - This should default to 3420 for mainnent and 13420 for floonet.
```

Comment:
should we leave this todo? :-)
perhaps just one line `if global::is_mainnet()`?

Reply:
Unfortunately not that simple - we haven't read the config in yet... so we can't yet access any global config.

Reply:
Tracking the underlying mainnet/floonet issue here - #2479.